processed 5 tasks

task 1 'publish'. lines 8-11:
created: object(1,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, storage_cost: 3967200,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2 'publish'. lines 13-18:
Error: Error checking transaction input objects: PackageVerificationTimedout { err: "Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: SuiMoveVerificationError, source: Some(\"0000000000000000000000000000000000000000000000000000000000000000::m::t. Invalid call to '0000000000000000000000000000000000000000000000000000000000000002::transfer::transfer' on an object of type '0x5b189794b7a1dcfbc81e43f2fec91eb84d3a1daae53131496f6573cc9557a7ff::m::S'. The transferred object's type must be defined in the current module. If the object has the 'store' type ability, you can use the non-internal variant instead, i.e. '0000000000000000000000000000000000000000000000000000000000000002::transfer::public_transfer'\"), command: None } }" }

task 3 'publish'. lines 20-25:
Error: Error checking transaction input objects: PackageVerificationTimedout { err: "Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: SuiMoveVerificationError, source: Some(\"0000000000000000000000000000000000000000000000000000000000000000::m::t. Invalid call to '0000000000000000000000000000000000000000000000000000000000000002::transfer::freeze_object' on an object of type '0x5b189794b7a1dcfbc81e43f2fec91eb84d3a1daae53131496f6573cc9557a7ff::m::S'. The transferred object's type must be defined in the current module. If the object has the 'store' type ability, you can use the non-internal variant instead, i.e. '0000000000000000000000000000000000000000000000000000000000000002::transfer::public_freeze_object'\"), command: None } }" }

task 4 'publish'. lines 27-32:
Error: Error checking transaction input objects: PackageVerificationTimedout { err: "Error executing ExecutionError: ExecutionError { inner: ExecutionErrorInner { kind: SuiMoveVerificationError, source: Some(\"0000000000000000000000000000000000000000000000000000000000000000::m::t. Invalid call to '0000000000000000000000000000000000000000000000000000000000000002::transfer::share_object' on an object of type '0x5b189794b7a1dcfbc81e43f2fec91eb84d3a1daae53131496f6573cc9557a7ff::m::S'. The transferred object's type must be defined in the current module. If the object has the 'store' type ability, you can use the non-internal variant instead, i.e. '0000000000000000000000000000000000000000000000000000000000000002::transfer::public_share_object'\"), command: None } }" }
